src/main/java/com/hermesworld/ais/galapagos/security/impl/AuditEventsListener.java

```diff
@@ -102,6 +102,16 @@ public AuditEventsListener(AuditEventRepository auditRepository) {
         return handleTopicEvent(event, "TOPIC_SUBSCRIPTION_APPROVAL_REQUIRED_FLAG_CHANGED");
     }
 
+    @Override
+    public CompletableFuture<Void> handleAddTopicProducers(TopicAddProducersEvent event) {
+        return FutureUtil.noop();
+    }
+
+    @Override
+    public CompletableFuture<Void> handleRemoveTopicProducer(TopicRemoveProducerEvent event) {
+        return FutureUtil.noop();
```

Comment:
MUST be logged! See other methods in this class on how to do this. Really important (audit log!)